com.unity.netcode.adapter.utp/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ All notable changes to this package will be documented in this file. The format
 
 ### Fixed
 
+- Lifted the limit of ~44KB for reliable payloads. Attempting to send a payload larger than that with reliable delivery would silently fail. Note that it is still not recommended to send such large reliable payloads, since their delivery could take a few network round-trips. (#1596)
```

Comment:
As written, it is not obvious if 
>Attempting to send a payload larger than that with reliable delivery would silently fail.
was the state before or after the fix. (well, to a non-native speaker, esp. because "would")

Reply:
Clarified the entry. Please let me know if it's still not obvious.